.gitignore

```diff
@@ -10,3 +10,4 @@ node_modules/
 .pnpm-debug.log
 .ps1-analyzer-stamp
 /TestSolutions/Lombiq.Analyzers.PowerShell.*/Violate-Analyzers.ps1
+/TestSolutions/*/.editorconfig
```

Comment:
The root one needs to be ignored too.

Reply:
I don't think a root one appeared for me, but okay.